bp_top/test/tb/bp_tethered/Makefile.verilator

```diff
@@ -40,7 +40,10 @@ $(SIM_DIR)/prog.elf: $(BP_TEST_MEM_DIR)/$(SUITE)/$(PROG).riscv
 $(SIM_DIR)/cce_ucode.mem: $(CCE_MEM_PATH)/$(CCE_MEM)
 	@cp $^ $@
 
-NBF_INPUTS = --ncpus=$(NCPUS) --ucode=cce_ucode.mem --mem=prog.mem --skip_zeros
+NBF_INPUTS = --ncpus=$(NCPUS) --ucode=cce_ucode.mem
+ifeq ($(PRELOAD_MEM_P), 0)
```

Comment:
Does this mean we can't preload mem using checkpoints? Is this a regression from before these changes?